package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gethue",
-  "version": "4.10.11",
+  "version": "4.11.12",
```

Comment:
Shouldn't it be `4.10.12`? 4.11 release is not out yet.

Reply:
Same at other places as well?

Reply:
Good catch, my bad.
Thanks.

Reply:
In-fact bumping the version to 5.0.0 as this is a change that break backward compatibility for whomever might be using druid & elasticsearch parsers.